src/main/java/de/florianmichael/viafabricplus/injection/mixin/fixes/minecraft/item/MixinHeldItemRenderer.java

```diff
@@ -51,16 +51,14 @@ private void transformLegacyBlockAnimations(AbstractClientPlayerEntity player, f
         }
 
         final Arm arm = hand == Hand.MAIN_HAND ? player.getMainArm() : player.getMainArm().getOpposite();
-
         if (blockHitAnimation) {
             applySwingOffset(matrices, arm, swingProgress);
-            matrices.translate(arm == Arm.RIGHT ? -0.14F : 0.14F, 0.12F, 0.12F);
-        } else {
-            matrices.translate(arm == Arm.RIGHT ? -0.15F : 0.15F, 0.07F, 0.12F);
         }
-        matrices.multiply(RotationAxis.POSITIVE_X.rotationDegrees(-102.25f));
-        matrices.multiply((arm == Arm.RIGHT ? RotationAxis.POSITIVE_Y : RotationAxis.NEGATIVE_Y).rotationDegrees(13.365f));
-        matrices.multiply((arm == Arm.RIGHT ? RotationAxis.POSITIVE_Z : RotationAxis.NEGATIVE_Z).rotationDegrees(78.05f));
+        
+        matrices.translate(-0.14142136F * (arm == Arm.RIGHT ? 1 : -1), 0.08F, 0.14142136F);
```

Comment:
It is questionable if the 1.7 block hit animation still looks correctly, also the operation for x can be simplified.

Reply:
The 1.7 item held values will be in the next pr

Reply:
Your changes are breaking 1.7, they need to get pushed together.

Reply:
ok gimme a second

Reply:
Actually nevermind, this change did not broke 1.7 blocking. Your other fix can go into a separate pull request.